bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/IterDarkObsForm.java

```diff
@@ -1,9 +1,9 @@
 package jsky.app.ot.editor;
 
-import javax.swing.*;
-import com.jgoodies.forms.factories.*;
-import com.jgoodies.forms.layout.*;
-import jsky.util.gui.*;
+		import javax.swing.*;
+		import com.jgoodies.forms.factories.*;
+		import com.jgoodies.forms.layout.*;
+		import jsky.util.gui.*;
```

Comment:
Hmm, what happened here with the indentation?

Reply:
Good question. I'll fix that. I wonder if IntelliJ did that.